lib/ui/view/organization/select_email/select_email_viewmodel.dart

```diff
@@ -63,5 +65,5 @@ class SelectEmailViewModel extends BaseViewModel {
     }
   }
 
-  void back() => _navigationService.popRepeated(1);
+  //void back() => _navigationService.popRepeated(1);
```

Comment:
Comment to be removed

Reply:
I didn't do tthis. I'll remove it anyway